tests/metrics/test_divergence.py

```diff
@@ -143,6 +138,54 @@ def test_divergence_single(self):
 
         pd.testing.assert_frame_equal(result, expected_df)
 
+    def test_divergence_duplicate(self):
+        """p3.data.divergence.duplicate"""
```

Comment:
Can you change this docstring, since `divergence_duplicate` is pretty vague? Perhaps even reference this PR?

Reply:
Fixed in a4c99137f30fc43e49a434885ae2ec5de70b2239.  I updated the docstrings for all the tests in this file, based on our discovery about what `unittest --verbose` prints.